src/meta/processors/admin/CreateSnapshotProcessor.cpp

```diff
@@ -69,9 +69,9 @@ void CreateSnapshotProcessor::process(const cpp2::CreateSnapshotReq&) {
 
     // step 3 : Create checkpoint for all storage engines and meta engine.
     auto csRet = Snapshot::instance(kvstore_,  client_)->createSnapshot(snapshot);
-    if (csRet != cpp2::ErrorCode::SUCCEEDED) {
+    if (csRet.isLeftType()) {
```

Comment:
The old snapshot and backup file could be removed?

Reply:
Yes, I am now calling the dropsnapshot interface to clean up temporary files when backup is done.

Reply:
I mean we could just remove the file `CreateSnapshotProcessor.cpp`? The `br` tool only call create backup?

Reply:
> I mean we could just remove the file `CreateSnapshotProcessor.cpp`? The `br` tool only call create backup?

I think it's okay to keep it for now, but we can remove CreateSnapshotProcessor.cpp when the BR is fully stabilized.